README.md

```diff
@@ -1 +1 @@
-# helloworld
\ No newline at end of file
+//I don't know how to write hello-world
```

Comment:
#start with this
#include <iostream>
using namespace std;
int main(){

return 0;
}